components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -1288,6 +1288,8 @@ public static GenericArtifact createAPIArtifactContent(GenericArtifact artifact,
             artifact.setAttribute(APIConstants.API_OVERVIEW_WEBSUB_SUBSCRIPTION_CONFIGURATION,
                     APIUtil.getWebsubSubscriptionConfigurationJsonFromDto(api.getWebsubSubscriptionConfiguration()));
 
+            artifact.setAttribute(APIConstants.API_OVERVIEW_WS_URI_MAPPING, APIUtil.getWsUriMappingJsonFromDto(api.getWsUriMapping()));
```

Comment:
this needs to be happening at apimgt.persistence implementation and is there any rxt level change for this?.

Reply:
RXT level change is already there in the master branch.

Reply:
And the CORS configurations are set here as well. So followed the same.

Reply:
This method is not used. Following is the usage of this method.

<img width="428" alt="Screenshot 2021-04-02 at 5 31 32 PM" src="https://user-images.githubusercontent.com/4861150/113414003-92804c00-93d9-11eb-8896-8aa753e7ca3b.png">

createAPI() method is not used when creating the API. check addAPI() method for this
updateWSDL methods are not used as well. that logic is in the updateAPI() method

Reply:
this content should be in addAPI() method in RegistryPersistenceImpl class

Reply:
please check the comment

Reply:
The relevant changes are already there in the RegistryPersistenceImpl class. Seems this is a redundant fix.